querysync/javatests/com/google/idea/blaze/qsync/AspectUnitTest.java

```diff
@@ -96,4 +96,15 @@ public void each_jar_belongs_to_a_single_target() throws IOException {
       assertWithMessage("targets for jar " + jarPath).that(jarToTarget.get(jarPath)).hasSize(1);
     }
   }
+
+  @Test
+  public void workspaceRootIncluded_getTargetSources() throws IOException {
+    ImmutableMap<Label, JavaTargetArtifacts> byTarget =
+        Maps.uniqueIndex(
+            JavaInfoTxt.WORKSPACE_ROOT_INCLUDED.readOnlyProto().getArtifactsList(),
+            t -> Label.of(t.getTarget()));
+    TestData target = TestData.WORKSPACE_ROOT_INCLUDED_QUERY;
+    // no sources should be there for a target within project scope.
```

Comment:
This might looks surprising, but I guess we assume we get it from `bazel query` so no need to get from the aspect.

FTR https://github.com/bazelbuild/intellij/pull/7039/files#diff-5a9bbef8c9fe9d68bc7a4d681f6335be8f023af28f5dbbd292dcdbd6d402c1c3R56